helm/azure-operator/Chart.yaml

```diff
@@ -6,3 +6,4 @@ home: "https://github.com/giantswarm/azure-operator"
 apiVersion: v1
 annotations:
   application.giantswarm.io/team: "celestial"
+  config.giantswarm.io/version: azure-operator
```

Comment:
This should be changed to `1.x.x` before merging the PR.

Reply:
can't you change it already?